webapp/cypress/integration/manageGroups.ts

```diff
@@ -21,4 +21,30 @@ describe('Manage groups', () => {
             blur()
         cy.get('.KanbanColumnHeader .Editable[value=\'Group 1\']').should('exist')
     })
+
+    it('MM-T4285 Adding group color', () => {
+        cy.visit('/')
+
+        cy.contains('+ Add board').click({force: true})
+        cy.contains('Empty board').click({force: true})
+        cy.contains('+ Add a group').click({force: true})
```

Comment:
@nishantwrp @saturninoabril why do we use `click` with `({force: true})`? This way any additional checks by Cypress (like that button is visible for example) are bypassed.

Reply:
I agree. Unfortunately, this is done in several places already. I'll submit a separate PR to clean it up which will include `@testing-library/cypress` that will also likely replace the use of `cy.contains` in favor of `cy.findByRole`, `cy.findByText`, etc.

Reply:
@saturninoabril I have already used `@testing-library/cypress` for #1826 so I will create a pull request soon. But I have not touched any other tests, they can be updated in a separate pull request like you have proposed.

P.S. And it is really non-trivial to use `cy.findByRole` because roles are not properly set for the rendered components and require changes in the source code of the webapp.

Reply:
Awesome, looking forward to it!

That's true and it will likely go on a separate epic whenever Boards go thru dealing with accessibility. However, I'd recommend to make use of it as much as possible only if applicable and makes sense, like in the case of button and input, ex.:
```javascript
cy.findByRole('button', {name: '+ Add a group'}).click()
```